adam-core/src/main/scala/org/bdgenomics/adam/converters/VariantContextConverter.scala

```diff
@@ -150,6 +153,74 @@ object VariantContextConverter {
       case GenotypeAllele.ALT                                => Allele.create(g.getVariant.getAlternateAllele)
     }
   }
+
+  private[adam] def cleanAndMixInSupportedLines(
```

Comment:
Yeah, it's an awkward method; I don't think there's a perfect home for it, but this seems like a more reasonable place than it's former home.